jekyll/_cci2/about-circleci.md

```diff
@@ -1,14 +1,18 @@
 ---
 layout: classic-docs
-title: "About CircleCI"
-description: "This document is a summary of continuous integration and how CircleCI enables engineering teams with automation. CircleCI automates your software builds, tests, and deployments."
-categories: [getting-started]
+title: About CircleCI
+description: This document is a summary of continuous integration and how CircleCI
+  enables engineering teams with automation. CircleCI automates your software builds,
+  tests, and deployments.
+categories:
+- getting-started
```

Comment:
some weirdness is happening here. Why the script changed 

```
categories: [getting-started]
```

to

```
categories:
- getting-started
```

Reply:
Please note the considerations section above. Some files are using 1 method of displaying the information and other files are using another. The script is just applying the same consistent rules to all as it rewrites the file.